srcalc/src/test/java/gov/va/med/srcalc/vista/RpcVistaPatientDaoTest.java

```diff
@@ -213,6 +213,11 @@ public final void testHealthFactorsValid() throws Exception
     public final void testNoHealthFactors()
     {
         final VistaProcedureCaller caller = mockVistaProcedureCaller();
+        when(caller.doRpc(
```

Comment:
I don't understand this change. How is it any different from what `mockVistaProcedureCaller()` already does?